src/vision/frc_vision_final.py

```diff
@@ -99,49 +111,52 @@ def unequal(new, old_list):
     # Find lines in selected image
     LINES = cv2.HoughLinesP(MASK_EDGES, 1, radians(.5), 25, maxLineGap=25)
 
+    # If there are lines:
     if LINES is not None:
         NUM_LINES = len(LINES)
         FILTERED_LINES = []
         X_TOTAL = 0
         Y_TOTAL = 0
+
         for NEW_LINE in LINES:
             x1, y1, x2, y2 = NEW_LINE[0]
             new_slope = degrees(np.arctan((y2 - y1)/(x2 - x1)))
+
+            # Checks if we have verified lines, and makes a new line based on that.
             if FILTERED_LINES:
-                if (new_slope < -40 or new_slope > 40) and unequal(new_slope, FILTERED_LINES):
-                    FILTERED_LINES.append(NEW_LINE)
-                    cv2.line(FILTERED_LINE_IMG, (x1, y1),
-                             (x2, y2), (0, 255, 0), 1)
-                    X_TOTAL += x1 + x2
-                    Y_TOTAL += y1 + y2
+                if (new_slope < -angleThreshold or new_slope > angleThreshold) and unequal(new_slope, FILTERED_LINES):
```

Comment:
This line is rather long, is it possible to shorten it, or break it up like the one after it?

Reply:
in range(-angleThreshold, angleThreshold)?

Reply:
Set `(new_slope < -angleThreshold or new_slope > angleThreshold)` to a variable and use that here and below